recipes-core/meta/package-index.bbappend

```diff
@@ -0,0 +1,4 @@
+# Normally no package index is created for the following DUMMYARCH but
+# we need one for building SDKs and meta-toolchain from feeds.
+PACKAGE_ARCHS:append = " sdk-provides-dummy-target"
+SDK_PACKAGE_ARCHS:append = " sdk-provides-dummy-${SDKPKGSUFFIX}"
```

Comment:
I'd like to just fix these being excluded from package-index builds altogether but that currently builds the indexes for values in `ALL_MULTILIB_PACKAGE_ARCHS` and `SDK_PACKAGE_ARCHS` and it's not immediately clear to me what adding these globally might impact elsewhere.

Reply:
Note that `PACKAGE_ARCHS` and `SDK_PACKAGE_ARCHS` are initially set in [`bitbake.conf`](https://github.com/ni/openembedded-core/blob/nilrt/master/hardknott/meta/conf/bitbake.conf).